modules/components/src/Aggs/TermAgg.js

```diff
@@ -62,6 +64,24 @@ class TermAggs extends React.Component {
 
   state = { showingMore: false, isExclude: false };
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
I kind of feel like the term agg search filter should be along the lines of `showExcludeOption`, basically a prop that can toggle on/off whether or not the term agg should show the text filter, and then it can pass the text input box into the AggsWrapper `filters` prop. thoughts?